experiments/react-native-uifabric/src/RNTester/FabricTester.tsx

```diff
@@ -19,70 +14,66 @@ msgq.spy(true);
 
 registerThemes();
 
-const styles = ReactNative.StyleSheet.create({
-  viewStyle: {
-    minHeight: 200,
-    justifyContent: 'space-between'
-  },
-  stackStyle: {
-    borderWidth: 2,
-    borderColor: '#bdbdbd',
-    padding: 12,
-    margin: 8
-  },
-  separatorStackStyle: {
-    height: 200,
-    flexDirection: 'row',
-    justifyContent: 'space-evenly'
+const EmptyComponent: React.FunctionComponent = () => {
+  return <Text fontSize={14} style={fabricTesterStyles.noTest}>Select a component from the left.</Text>;
+}
+
+// sort tests alphabetically by name 
+const sortedTestComponents = allTestComponents.sort(
+  (a, b) => a.name.localeCompare(b.name)
+);
+
+const TestListSeparator = Separator.customize(
+  {
+    tokens: {
+      color: 'darkGray',
+      separatorWidth: 2
+    }
   }
-});
+);
 
-const Panel: React.FunctionComponent = () => {
-  const [disabled, setDisabled] = React.useState(false);
-  const onClick = React.useCallback(() => setDisabled(!disabled), [disabled, setDisabled]);
-  const theme = useTheme();
-  return (
-    <ReactNative.View style={[styles.viewStyle, styles.stackStyle, { backgroundColor: theme.colors.background }]}>
-      <PrimaryButton onPress={onClick} content="Primary Button" disabled={disabled} />
-      <Button onPress={onClick} content="Default Button" disabled={disabled} />
-      <StealthButton onPress={onClick} content="Stealth Button" disabled={disabled} />
-      <Text>This is a text element</Text>
-      <Button onPress={onClick} content="This button has longer text" disabled={disabled} />
-    </ReactNative.View>
+export interface IFabricTesterProps {

```

Comment:
You don't actually need to export this right?

Reply:
nvm, though what's the expected usage of RNTesterApp? How are we supposed to pass props in?

Reply:
I'm making changes to ReactTest in devmain to make it possible to pass in initialProps to plugins. Will send out a CR later today.